src/data_utils.py

```diff
@@ -93,14 +157,20 @@ def _fetch_and_push_files(self, data_dir, file_paths: list, file_pattern: str):
 
 if __name__ == "__main__":
     """Testing Area"""
+    tokenizer = Wav2Vec2Processor(is_tokenizer=True)
 
-    tr_dataset = DataLoader(data_dir="../data/librispeech/test-clean")(shuffle=True)
+    tr_data_args = LibriSpeechDataLoaderArgs(data_dir="../data/librispeech/test-clean")
```

Comment:
Is this for testing purposes? Because we'd want to specify the paths in a separate script, right?

Reply:
Yes, this is just for testing. It will be removed once model training is done.